### PR TITLE
Writing examples for amazon Elastic Beanstalk (PaaS)

### PR DESCRIPTION
**Pull request by rajitha1998**
Related Issue #41
## Proposed Changes

  - Writing examples for Amazon Elastic Beanstalk (platform as a service)

### CODE REVIEW

**Comment by rajikaimal** [comment]
Let's give an equivalent name for this. AWS it's BeanStalk and in Azure it's App Service if I'm not wrong. In the context of NodeCloud let's try to find a common ground and give a sensible name.

This means we want to change the APIs exposed in AWS beanstalk implementation in nodecloud-aws-plugin. (as well as App Service in Azure plugin)

**Comment by rajitha1998** [comment]
> Let's give an equivalent name for this. AWS it's BeanStalk and in Azure it's App Service if I'm not wrong. In the context of NodeCloud let's try to find a common ground and give a sensible name.
> 
> This means we want to change the APIs exposed in AWS beanstalk implementation in nodecloud-aws-plugin to change. (as well as App Service in Azure plugin)

Yes in Azure it's App service and in Google cloud it's app engine.  I will rename it 👍

**Comment by rajitha1998** [comment]
Shall we name it as PaaS? Since Beanstalk, App engine and App service provides platform as the service. @rajikaimal

**Comment by rajikaimal** [comment]
Yep, makes sense

**Comment by rajitha1998** [comment]
Updated the name @rajikaimal. The relevant [PR](https://github.com/leopardslab/nodecloud-aws-plugin/pull/31) for the plugin is also added.

File: examples/compute/elasticBeanstalk.js [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+    result => {
+      console.log("Output : ", result);
+    },
+    error => {
+      console.error("Error :", error);
+    }
+  );
+}
+
+function getApplicatios() {
```
Comment (rajikaimal) [inline examples/compute/elasticBeanstalk.js]: Typo

Comment (rajitha1998) [reply examples/compute/elasticBeanstalk.js]: @rajikaimal  Corrected it :)

**Review by rajikaimal — APPROVED** [review]

**Pull request merged**